satellite/metainfo/metainfo.go

```diff
@@ -523,12 +524,11 @@ func (endpoint *Endpoint) filterValidPieces(ctx context.Context, pointer *pb.Poi
 			return Error.New("all pieces needs to have the same size")
 		}
 
-		// we repair when the number of healthy files is less than or equal to the repair threshold
-		// except for the case when the repair and success thresholds are the same (a case usually seen during testing)
-		if int32(len(remotePieces)) <= remote.Redundancy.RepairThreshold && int32(len(remotePieces)) < remote.Redundancy.SuccessThreshold {
```

Comment:
do we still want to return a distinct error if the number of pieces is less than the repair threshold? Or do we want to return the same error in either case as you are doing now?

Reply:
yeah I think it would make more sense to add a distinct error for less than repair threshold. I'll go ahead and add that.

Reply:
I think we can remove the entire block above this.  Repair threshold _should_ always be less than success threshold.  If we need a sanity check for this, that sanity check should probably be elsewhere.